src/pages/Staking/AuthorizeStakingApps/index.tsx

```diff
@@ -114,6 +114,11 @@ const AuthorizeStakingAppsPage: FC = () => {
     },
   }
 
+  const isFullyAuthorized = Object.values(appsAuthData).every(
+    ({ status }) =>
+      status === "authorized" || status === "authorization-not-required"
+  )
```

Comment:
We need to modify this condition. A stake only requires 40k of its tokens authorized to be considered 'authorized.' If we have a stake of 100k T and authorize only 40k T, it will still be labeled 'authorized,' but in reality, it's not fully authorized.

Since it is mentioned in [the issue](https://github.com/threshold-network/token-dashboard/issues/573) that the info bubble should be hidden only after the app is 100% authorized, we should refine this condition accordingly:

```suggestion
  const isFullyAuthorized = Object.values(appsAuthData).every(
    ({ status, percentage }) =>
      (status === "authorized" && percentage === 100) ||
      status === "authorization-not-required"
  )
```

Reply:
c8a8577cf041c04bb411544ba715c03977d275e0